rules/linux/persistence_linux_shell_activity_by_web_server.toml

```diff
@@ -17,9 +17,9 @@ false_positives = [
 ]
 from = "now-9m"
 index = ["auditbeat-*", "logs-endpoint.events.*", "endgame-*"]
-language = "kuery"
+language = "eql"
```

Comment:
We had a discussion of  https://github.com/elastic/detection-rules/issues/1730
And this was something that was brought and made aware type changing is not an option.